packages/balancer/agents/autonomous_fund/tests/test_contracts/test_managed_pool/test_contract.py

```diff
@@ -158,6 +128,10 @@ def test_get_gradual_weight_update_params(self) -> None:
             == INITIAL_POOL_WEIGHTS
         )
 
+    def teardown_class(self) -> None:
```

Comment:
Please check you're not overriding `super`'s `teardown_class` method but extend it

Reply:
None of the super classes have `teardown_class`. None of them are test classes.